src/pages/en/guides/rss.mdx

```diff
@@ -141,35 +141,41 @@ items: import.meta.glob('./blog/*.{md,mdx}'),
 
 The `content` key contains the full content of the post as HTML. This allows you to make your entire post content available to RSS feed readers.
 
+:::caution
```

Comment:
Gonna call a flag on the play for caution here! It's not a good look to basically have Section Title then immediately CAUTION as if the *thing itself* is risky. If it's only our examples/explanation that requires a warning, then we fix that with writing!  ;)

There are a couple of different ways we could go about doing this, but let's start by getting your gut reaction: if you were not allowed to use a caution, AND not allowed to put ANY kind of aside (note/tip) upfront like this, how/where would include this piece of information?